src/mmw/js/src/analyze/templates/loading.html

```diff
@@ -0,0 +1,8 @@
+<div class="analyze-message-inner">
+    <div>
+        <i class="fa fa-circle-o-notch fa-spin"></i>
+    </div>
+    <div class="analyze-message-text">
+        Loading Data
```

Comment:
I realize this is the message in the mock-ups, but we may want to switch it to something like "Analyzing" or something more descriptive.